tox.ini

```diff
@@ -53,7 +53,19 @@ labels =
 change_dir =
     unit: tests/unit
     integration: tests/integration
-commands = pytest {tty:--color=yes} --cov --cov-report=xml:{tox_root}/results/coverage-{env_name}.xml --junit-xml={tox_root}/results/test-results-{env_name}.xml {posargs}
+commands =
+    unit: pytest {tty:--color=yes} --cov --cov-report=xml:{tox_root}/results/coverage-{env_name}.xml --junit-xml={tox_root}/results/test-results-{env_name}.xml {posargs}
+    integration: pytest {tty:--color=yes} --cov --cov-report=xml:{tox_root}/results/coverage-{env_name}.xml --junit-xml={tox_root}/results/test-results-{env_name}.xml -m "not smoketest" {posargs}
```

Comment:
how come we don't want to run the smoketest in non-macos?

Reply:
Because that smoketest only for macos? The integration tests in linux usually works fine.